httpx/_client.py

```diff
@@ -164,6 +164,7 @@ class BaseClient:
     def __init__(
         self,
         *,
+        request_class: type[Request] | None = None,
```

Comment:
Presumably also `response_class`, right?
Also I guess move this to the end of the parameters.

Reply:
> Presumably also response_class, right?

I thought about that, but baseclient has no logic related with responses, so response_class is now added only in Client for passing to transports. I can added response_class, if needed.